### PR TITLE
AUTO-112: Set up Verify MSA on PaaS

### PR DESCRIPTION
**Pull request by adityapahuja**
This pull requests contains multiple commits required for setting up Verify MSA on PaaS. Please see commits' messages for further details.

### CODE REVIEW

**Comment by Krenair** [comment]
Can't review Andy's first change but this looks good otherwise

File: ci/manifest.yml.tmpl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      TRUSTSTORE_PASSWORD: $TRUSTSTORE_PASSWORD
+      EUROPEAN_IDENTITY_ENABLED: true
+      HUB_CONNECTOR_ENTITY_ID: ${HUB_CONNECTOR_ENTITY_ID}
+      TRUST_ANCHOR_URI: https://$SIGNIN_DOMAIN/SAML2/metadata/trust-anchor
+      METADATA_SOURCE_URI: https://$SIGNIN_DOMAIN/SAML2/metadata/aggregator
+      ENCRYPTION_KEY: $ENCRYPTION_KEY
+      ENCRYPTION_CERT: $ENCRYPTION_CERT
+      SIGNING_KEY: $SIGNING_KEY
+      SIGNING_CERT: $SIGNING_CERT
+      ENVIRONMENT: $APP_ENV
```
Comment (philandstuff) [inline ci/manifest.yml.tmpl]: we need to make sure that `$APP_ENV` and `$SIGNIN_DOMAIN` are set when template is instatiated - we saw issues today that they were not causing MSA breakage :(

Comment (Krenair) [reply ci/manifest.yml.tmpl]: They are set when the template is instantiated: https://github.com/alphagov/verify-infrastructure-config/pull/179/commits/c30f1347866e05c1c4ed96ae1a5292fd59289e8a

Comment (andy-paine) [reply ci/manifest.yml.tmpl]: We got to the bottom of this - the changes to the pipeline weren't applied so the parameters weren't available yet. Now changes have been applied it looks good

Comment (Krenair) [reply ci/manifest.yml.tmpl]: So I think what went wrong here is I had forgotten to set-pipeline after the above linked commit, this branch got deployed without that important change and this caused the staging problem 2 days ago.

**Pull request merged**